tests/test_custom_models.py

```diff
@@ -230,6 +230,8 @@
     ("Conv2d 2 LOHA", "Conv2d", LoHaConfig, {"target_modules": ["conv2d", "lin0"]}),
     ("Conv2d 3 LOHA", "Conv2d", LoHaConfig, {"target_modules": ["conv2d"], "use_effective_conv2d": True}),
     ("Conv2d 4 LOHA", "Conv2d", LoHaConfig, {"target_modules": ["conv2d", "lin0"], "use_effective_conv2d": True}),
+    ("Conv1D LoHa", "Conv1d", LoHaConfig, {"target_modules": ["conv1d"]}),
```

Comment:
```suggestion
    ("Conv1d LOHA", "Conv1d", LoHaConfig, {"target_modules": ["conv1d"]}),
```

to not confuse this with `Conv1D` from transformers and for consistency with other tests.